### PR TITLE
[User] Submit bill and purchase

### PR DESCRIPTION
**Pull request by minhld99**
## Related Tickets
- [#39973](https://edu-redmine.sun-asterisk.vn/issues/39973)

## WHAT
- Fake thanh toán sử dụng Razorpay

## HOW
- N/A

## WHY
- N/A

## Evidence (Screenshot or Video)

- Fake payment

image

- On success

image

- Order history updated

image

### CODE REVIEW

**Comment by minhld99** [comment]
ready

File: main/static/js/script.js [hunk 1]
```diff
@@ -497,4 +497,52 @@ $(document).ready(function(){
...
+                    "image": "/static/img/logo.jpeg",
+                    "order_id": `${rs.rp_order_id}`, 
+                    "callback_url": `${rs.callback_url}`,
+                    "prefill": {
+                        "name": `${order.recipient}`,
+                        "email": `${rs.email}`,
+                        "contact": `+94${order.phone_number}`,
+                    },
+                    "notes": {
+                        'Shipping address': `${order.address}`,
```
Comment (huongnt-2545) [inline main/static/js/script.js:531]: `'Shipping address'` => `shipping_address`

File: main/views.py [hunk 2]
```diff
@@ -349,3 +352,79 @@ def cancel_order(request):
...
+    if request.method == "POST":
+        language = request.POST.get('lang')
+        order_id = request.POST.get('order_id')
+        current_order = get_object_or_404(Bill, id=order_id)
+        order_total = current_order.total
+        order_currency = 'USD'
+        callback_url = f'http://localhost:8000{language}handle-payment/'
+        order_receipt = order_id
+        notes = {
+            'Recipient': current_order.recipient,
```
Comment (huongnt-2545) [inline main/views.py:367]:
```suggestion
            'recipient': current_order.recipient,
            'shipping_address': current_order.address,
            ...
```

**Comment by minhld99** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
merged

**Pull request merged**